docs/pywbemlistener/index.rst

```diff
@@ -20,15 +20,15 @@ Pywbemlistener command
 
 The pywbemlistener command can run and manage WBEM listeners.
 
-Each listener is a process that executes the `pywbemlistener run`
+Each listener is a process that executes the ``pywbemlistener run``
 command to receive WBEM indications sent from a WBEM server.
 
-A listener process can be started with the `pywbemlistener start`
-command and stopped with the `pywbemlistener stop` command.
+A listener process can be started with the ``pywbemlistener start``
```

Comment:
Should we be discussing the run command here also and the fact that the difference between the run and the start is whether the indication listener is a subprocess of the initiating pywbemlistener.

Reply:
I don't think so. We do not want users to use it directly, and this is just the overview here.

Reply:
I still think somewhere we should state these differences and why the run command is public.  As it is, the user is faced with run and start and the first question is which do I use.